def system_out(exception: Exception):
    print(exception)
    exit()